packages/core-modules/contracts/interfaces/IElectionModule.sol

```diff
@@ -20,15 +20,23 @@ interface IElectionModule {
 
     function setSeatCount(uint seats) external;
 
-    function setEpochDuration(uint duration) external;
+    function setEpochDuration(uint64 duration) external;
 
     function setPeriodPercent(uint8 percent) external;
 
     function setNextSeatCount(uint seats) external;
 
-    function setNextEpochDuration(uint duration) external;
+    function setNextEpochDuration(uint64 duration) external;
 
     function setNextPeriodPercent(uint8 percent) external;
 
     function elect(address[] memory candidates) external;
+
+    function epochFinised() external view returns (bool);
+
+    function isEpochNominationPeriod() external view returns (bool);
```

Comment:
These two functions `isEpochNominationPeriod` and `isEpochVotingPeriod` could have better naming.
Do you mean `isOnEpochNominationPeriod` and `isOnEpochVotingPeriod`?

If so, I'd recommend `isNominating` and `isVoting`.

Reply:
I see you renamed one of the functions to `isNomination`. Is this a typo? My suggestion was `isNominating`.

Reply:
fixed